src/libcore/option.rs

```diff
@@ -967,7 +969,7 @@ impl<T> Option<T> {
     /// assert_eq!(x.zip_with(y, Point::new), Some(Point { x: 17.5, y: 42.7 }));
     /// assert_eq!(x.zip_with(None, Point::new), None);
     /// ```
-    #[unstable(feature = "option_zip", issue = "70086")]
+    #[unstable(feature = "option_zip_with", issue = "70086")]
```

Comment:
When we stabilize only a subset of a library feature, we change the name of the stabilized part, not the still unstable part. That way people only get unused feature warnings downstream, not build failures.

Reply:
I changed the stabilized feature to `option_zip_option`.